.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/_nuts/github.com/ThomsonReutersEikon"]
+	path = vendor/_nuts/github.com/ThomsonReutersEikon
+	url = https://github.com/ThomsonReutersEikon/go-ntlm
```

Comment:
Is there any reason this is a submodule instead of just being in the Nut.toml and added directly like the other dependencies? Submodules tend to over-complicate things unless you really need them.